src/integrationTest/java/uk/gov/hmcts/darts/testutils/stubs/UserAccountStub.java

```diff
@@ -186,4 +188,14 @@ public UserAccountEntity createExternalUser(String guid, SecurityGroupEntity sec
         return testUser;
     }
 
+    public UserAccountEntity createAdminUser() {
+        var adminGroup = securityGroupRepository.findByGroupName(SecurityRoleEnum.ADMIN.name())
```

Comment:
This looks to me like it should be changed to be something like `findBySecurityGroupRole`? We use the security role enum id as it is a standing data table - security_role(rol_id). Since it's a test and we know that we have  a security_group with group_name = ADMIN - I guess it could just be a String constant here?

Reply:
Good spot, yeah I think `"ADMIN"` as a string literal is a reasonable approach here. If for some reason that name is changed, the associated tests will fail and should be quite obvious as to why.